developer/examples/build.vpc.windows.pkr.hcl

```diff
@@ -17,19 +17,39 @@ variable "ansible_inventory_file" {
   default = "${env("ANSIBLE_INVENTORY_FILE")}"
 }
 
+variable "subnet_id" {
```

Comment:
Packer Template variables should be introduce as plain text on the template. I know it is reasonable to think about store them on the env file, however, there is no need to store them as environment variables. Environment variables meaning is to store sensitive data like API keys, or Ansible and Packer configuration variables due you cannot publish them on a public repo for security reasons. The main reason behind is we want to keep industry standards, so clients does not have any problem when they move from other cloud providers (i.e. AWS Packer plugin)